### PR TITLE
Fish 8992 run activation tck

### PR DESCRIPTION
**Pull request by luiseufrasio**
The build finishes with success, but it seems that no ant tests are found

### CODE REVIEW

File: activation-tck/pom.xml [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+        <payara.home>${project.build.directory}/payara7</payara.home>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>fish.payara.distributions</groupId>
+            <artifactId>payara</artifactId>
+            <version>${payara.version}</version>
+            <type>zip</type>
+        </dependency>
```
Comment (Pandrex247) [inline activation-tck/pom.xml:70]: Defined in parent pom `payara-server-managed` profile
```suggestion
```

File: activation-tck/pom.xml [hunk 2]
```diff
@@ -0,0 +1,232 @@
...
+    <artifactId>activation-tck</artifactId>
+    <packaging>pom</packaging>
+
+    <name>TCK: Activation</name>
+
+    <properties>
+        <ant.home>${project.build.directory}/apache-ant-${ant.version}</ant.home>
+        <ant.zip.url>https://archive.apache.org/dist/ant/binaries/apache-ant-${ant.version}-bin.zip</ant.zip.url>
+        <tck.home>${project.build.directory}/activation-tck</tck.home>
+        <payara.home>${project.build.directory}/payara7</payara.home>
```
Comment (Pandrex247) [inline activation-tck/pom.xml:61]: Defined in parent pom
```suggestion
```

File: activation-tck/pom.xml [hunk 3]
```diff
@@ -0,0 +1,232 @@
...
+                        <goals>
+                            <goal>unpack-dependencies</goal>
+                        </goals>
+                        <configuration>
+                            <includeArtifactIds>payara,jakarta-activation-tck</includeArtifactIds>
+                            <outputDirectory>${project.build.directory}</outputDirectory>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
```
Comment (Pandrex247) [inline activation-tck/pom.xml:122]: Unpacking Payara defined in `payara-server-managed` profile
```suggestion
            <!-- Unpack TCK -->
            <plugin>
                <artifactId>maven-dependency-plugin</artifactId>
                <configuration>
                    <skip>${skipTests}</skip>
                </configuration>
                <executions>
                    <execution>
                        <id>unpack-tck</id>
                        <phase>pre-integration-test</phase>
                        <goals>
                            <goal>unpack-dependencies</goal>
                        </goals>
                        <configuration>
                            <includeArtifactIds>jakarta-activation-tck</includeArtifactIds>
                            <outputDirectory>${project.build.directory}</outputDirectory>
                        </configuration>
                    </execution>
                </executions>
            </plugin>
```

File: activation-tck/pom.xml [hunk 4]
```diff
@@ -0,0 +1,232 @@
...
+                    <execution>
+                        <id>download-ant</id>
+                        <phase>generate-resources</phase>
+                        <goals>
+                            <goal>wget</goal>
+                        </goals>
+                    </execution>
+                </executions>
+                <configuration>
+                    <skip>${skipITs}</skip>
```
Comment (Pandrex247) [inline activation-tck/pom.xml:96]:
```suggestion
                    <skip>${skipTests}</skip>
```

File: activation-tck/pom.xml [hunk 5]
```diff
@@ -0,0 +1,232 @@
...
+                    <unpack>true</unpack>
+                    <outputDirectory>${project.build.directory}</outputDirectory>
+                </configuration>
+            </plugin>
+
+            <!-- Unpack Payara and TCK -->
+            <plugin>
+                <artifactId>maven-dependency-plugin</artifactId>
+                <configuration>
+                    <skip>${skipITs}</skip>
```
Comment (Pandrex247) [inline activation-tck/pom.xml:107]:
```suggestion
                    <skip>${skipTests}</skip>
```

File: activation-tck/pom.xml [hunk 6]
```diff
@@ -0,0 +1,232 @@
...
+                        <exclusions>
+                            <exclusion>
+                                <groupId>ant</groupId>
+                                <artifactId>ant</artifactId>
+                            </exclusion>
+                        </exclusions>
+                    </dependency>
+                </dependencies>
+                <configuration>
+                    <skip>${skipITs}</skip>
```
Comment (Pandrex247) [inline activation-tck/pom.xml:146]:
```suggestion
                    <skip>${skipTests}</skip>
```

**Review by Pandrex247 — CHANGES_REQUESTED** [review]

**Comment by breakponchito** [comment]
@luiseufrasio please review comments added by @Pandrex247

**Review by Pandrex247 — APPROVED** [review]

**Pull request merged**